js/PDBLoaderImproved.js

```diff
@@ -358,19 +371,18 @@ catch(e){
 				{
 					// don't bond to other chains
 					var neighbor = molecule.atoms[j];
-					if(neighbor.chainId !== atom.chainId
-						|| Math.abs(neighbor.seqRes - atom.seqRes) > 1
-					)
-						continue;
-
-					var v2 = new THREE.Vector3(neighbor.x, neighbor.y, neighbor.z);
+					if(neighbor.chainID !== atomChainID) { continue; }
 
 					// get distance between atoms, compared to covalent radii
-					var dist = mesh.position.distanceTo(v2);
-					var covalentDist = 0.01*covalentRadius[e] + 0.01*covalentRadius[neighbor.element.toLowerCase()];
+					var dist = mesh.position.distanceToSquared(neighbor);
```

Comment:
We don't need the actual distance, since we're only using it for comparison. distanceToSquared avoids a square root operation.